crates/testlang/outputs/npm/package/package.json

```
@@ -0,0 +1,78 @@
+{
+  "name": "@slang-private/slang-testlang",
+  "version": "0.13.1",
```

To avoid publishing this package, I suggest removing the `publishConfig` entry, and adding `private: true`:

```suggestion
  "version": "0.13.1",
  "private": true,
```

---

Done